playbooks/common/openshift-cluster/additional_config.yml

```diff
@@ -24,7 +24,3 @@
   - role: cockpit
     when: not openshift.common.is_atomic and ( deployment_type in ['atomic-enterprise','openshift-enterprise'] ) and
       (osm_use_cockpit | bool or osm_use_cockpit is undefined )
-  - role: flannel_register
```

Comment:
One question that I've had with the flannel config, is that it appears that there is a single point of failure, with just the "first master" running the flannel_register role, is this something that can also be configured on the other masters, or is there additional work needed to make the flannel integration ha?